.golangci.yml

```diff
@@ -18,6 +18,10 @@ linters:
           deny:
             - pkg: github.com/opencontainers/runc
               desc: We don't want to depend on runc (libcontainer), unless there is no other option; see https://github.com/opencontainers/runc/issues/3028.
+    forbidigo:
+      forbid:
+        - pkg: ^regexp$
+          msg: Use internal/lazyregexp.New instead.
```

Comment:
Can you

- include this fix in the `ci: bump golangci from 6.5.2 to 7.0.0` commit
- move the `ci: bump golangci from 6.5.2 to 7.0.0` to be the last commit (after all the other changes)?